### PR TITLE
issue: 875772 Add --enable-opt-log configuration option

### PR DESCRIPTION
**Pull request by igor-ivanov**
see related master: https://github.com/Mellanox/libvma/commit/2c2c665a7cd361fd8c4a30cf06a04d9d916bd892

@OphirMunk please look at

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/544/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/551/ for details (Mellanox internal link).

File: src/vlogger/vlogger.h [hunk 1]
```diff
@@ -60,27 +60,53 @@
 #define __log_err(log_fmt, log_args...)          do { VLOG_PRINTF(VLOG_ERROR, log_fmt, ##log_args); } while (0)
 #define __log_warn(log_fmt, log_args...)         do { VLOG_PRINTF(VLOG_WARNING, log_fmt, ##log_args); } while (0)
 #define __log_info(log_fmt, log_args...)         do { VLOG_PRINTF(VLOG_INFO, log_fmt, ##log_args); } while (0)
+
+#if defined(VMA_OPTIMIZE_LOG)
+#define __log_details(log_fmt, log_args...)      ((void)0)
```
Comment (OphirMunk) [inline src/vlogger/vlogger.h]: @igor-ivanov  If --log_details is not on fast path - please consider not excluding it

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/546/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/553/ for details (Mellanox internal link).

**Comment by OphirMunk** [comment]
Merged #225

**Pull request closed without merging**